### PR TITLE
🔥 Remove System.Drawing dependency from main library

### PR DESCRIPTION
**Pull request by paulcscharf**
## ✨ What's this?
What the title says. The only thing that was left remaining was the FontFactory, which as it turns out I could just move to the System.Drawing library without any issues.

Next steps will include similar implementations with ImageSharp and/or SkiaSharp (depending which I can figure out easily).

## 🔍 Why do we want this?
Better Xplat support.

### 💥 Breaking changes
Removed FontFactory from the main library, a dependency on Bearded.Graphics.System.Drawing library is now needed.

### CODE REVIEW

File: Bearded.Graphics/Bearded.Graphics.csproj [hunk 1]
```diff
@@ -10,12 +10,11 @@
 
   <ItemGroup>
     <PackageReference Include="Bearded.Utilities" Version="0.2.0.393-dev" />
-    <PackageReference Include="OpenTK.Graphics" Version="4.4.0" />
-    <PackageReference Include="OpenTK.Windowing.Common" Version="4.4.0" />
-    <PackageReference Include="OpenTK.Windowing.Desktop" Version="4.4.0" />
-    <PackageReference Include="OpenTK.Windowing.GraphicsLibraryFramework" Version="4.4.0" />
```
Comment (paulcscharf) [inline Bearded.Graphics/Bearded.Graphics.csproj:16]: Needed to update either these or the reference in Bearded.Graphics.System.Drawing to make the example work. Felt wrong to downgrade the other so I upgraded this one.

After merging this, I'll submit another that pushes the version to 4.7.5 for all projects (latest stable).

Comment (tomrijnbeek) [reply Bearded.Graphics/Bearded.Graphics.csproj:16]: Probably dependabot will start picking up these changes now that the solution file is up to date again. You can always force a check on that as well.

**Review by tomrijnbeek — APPROVED** [review]
I didn't review the font factory file because I assume it was a straight copy, even though GitHub doesn't show it that way.

**Comment by paulcscharf** [comment]
Straight copy with some namespace changes yeah - my bad for also changing indentation of the whole file at the same time.

**Pull request merged**